### DISCUSSION TITLE
Can add person group list for divide a large number of person?

### DISCUSSION THREAD

**Discussion by win2ter** [immich-app/immich#12679] [Feature Request]
### The bug

Can add person group list for divide a large number of person?

My photo album has identified 771 people, and the attributes of the people (classmates, family members, friends, colleagues) are mixed with each other, making it difficult to organize and find.

Since photos and AI recognition will generate a lot of avatars of people, it is not convenient to view photos of related people.

Could you please add custom character groups to classify each character, such as family/friends/classmates/company A/company B.

### The OS that Immich Server is running on

NAS

### Version of Immich Server

v1.115.0

### Version of Immich Mobile App

v1.115.0

### Platform with the issue

- [X] Server
- [X] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
-
```

### Your .env content

```Shell
-
```

### Reproduction steps

My photo album has identified 771 people, and the attributes of the people (classmates, family members, friends, colleagues) are mixed with each other, making it difficult to organize and find.

**Comment by bo0tzz** [comment]
#6079
